reviewing/outdated/src/createManifestGetter.ts

```diff
@@ -28,10 +36,31 @@ export async function getManifest (
   packageName: string,
   bareSpecifier: string
 ): Promise<DependencyManifest | null> {
-  const resolution = await resolve({ alias: packageName, bareSpecifier }, {
-    lockfileDir: opts.lockfileDir,
-    preferredVersions: {},
-    projectDir: opts.dir,
-  })
-  return resolution?.manifest ?? null
+  let publishedBy: Date | undefined
+  if (opts.minimumReleaseAge) {
+    const isExcluded = opts.minimumReleaseAgeExclude && createMatcher(opts.minimumReleaseAgeExclude)(packageName)
+    if (!isExcluded) {
+      publishedBy = new Date(Date.now() - opts.minimumReleaseAge * 60 * 1000)
+    }
+  }
+
+  // When minimumReleaseAge is set and requesting 'latest',
+  // use '*' instead to get the latest version that meets the age requirement
+  const specifier = (publishedBy && bareSpecifier === 'latest') ? '*' : bareSpecifier
```

Comment:
I don't understand why this is needed. We repopulate "latest" after filtering out versions

Reply:
Ah, you're right, this was unnecessary. 
It could also resolve versions from other dist-tags besides `latest`, which wasn't the intended behavior.

Fixed in this commit.
[fix: remove unnecessary 'latest' to '*' conversion in outdated command](https://github.com/pnpm/pnpm/pull/10030/commits/bb5d4dfae348290147a404645cfde051331a8b0a)